webui/react/package.json

```diff
@@ -97,6 +97,7 @@
     "@types/express": "^4.17.17",
     "@types/humanize-duration": "^3.27.1",
     "@types/js-yaml": "^4.0.5",
+    "@types/lodash": "^4.14.195",
```

Comment:
installed missing lodash type lib

Reply:
I see that we have lodash listed in our dependencies, but I can't find any places where we actually use it. I think this warrants further discussion.